frontend/packages/db/prisma/migrations/20250328105323_add_branch_name_to_knowledge_suggestion/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
+  Warnings:
+
+  - Added the required column `branchName` to the `KnowledgeSuggestion` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "KnowledgeSuggestion" ADD COLUMN     "branchName" TEXT NOT NULL;
```

Comment:
KnowledgeSuggestion table is empty in production now👍🏻 

<img width="1165" alt="Screenshot 2025-03-28 at 20 59 05" src="https://github.com/user-attachments/assets/74c00e03-0380-4614-ab66-f42a46e2c21a" />

ref: https://supabase.com/dashboard/project/vsdfxfkorhtkxqcpczyr/editor/52884?schema=public